src/service/viewport-impl.js

```diff
@@ -1086,22 +1122,43 @@ export class ViewportBindingNaturalIosEmbed_ {
   }
 
   /** @override */
-  updatePaddingTop(paddingTop, opt_updateScrollPos, opt_lastPaddingTop) {
+  hideViewerHeader(lastPaddingTop) {
+    this.removePaddingTop_(true, lastPaddingTop);
+  }
+
+  /** @override */
+  showViewerHeader(paddingTop) {
```

Comment:
I changed the behavior of `showViewerHeader` in natural binding. It could not be inlined after the change